addons/payment_adyen/utils.py

```diff
@@ -53,15 +53,21 @@ def format_partner_address(partner):
     """
     STREET_FORMAT = '%(street_number)s/%(street_number2)s %(street_name)s'
     street_data = split_street_with_params(partner.street, STREET_FORMAT)
-    return {
+    state_or_province = partner.state_id.code
```

Comment:
It's the same but cognitively easier to read: think about it once and then you can just trust the variable name.
```suggestion
    state_or_province = partner.state_id.code or ''
```